metricbeat/module/docker/vendor/github.com/fsouza/go-dockerclient/container.go

```diff
@@ -99,6 +99,21 @@ func (p Port) Proto() string {
 	return parts[1]
 }
 
+// HealthCheck represents one check of health.
+type HealthCheck struct {
```

Comment:
The commit id must be updated here: https://github.com/elastic/beats/blob/master/metricbeat/module/docker/glide.yaml Please make sure to copy all changes from this commit to the vendor directory.